javaslang/src/test/java/javaslang/collection/AbstractSortedSetTest.java

```diff
@@ -16,6 +16,12 @@
 public abstract class AbstractSortedSetTest extends AbstractSetTest {
 
     @Override
+    abstract protected <T> SortedSet<T> empty();
+
+    @Override
+    abstract protected <T> SortedSet<T> emptyWithNull();
```

Comment:
\[Checkstyle\] ERROR: 'protected' modifier out of order with the JLS suggestions\.